api/v1/paths/notes/{id}.js

```diff
@@ -35,7 +49,8 @@ const del = async (req, res) => {
   }
 };
 
+get.apiDoc = paths['/notes/{noteID}'].get;
 patch.apiDoc = paths['/notes/{noteID}'].patch;
 del.apiDoc = paths['/notes/{noteID}'].del;
 
-module.exports = { patch, del };
+module.exports = { get, patch, del };
```

Comment:
Same thing here.

Reply:
From our discussion, we're only going to require properties on their own line once there are 4 or more properties.